ui/selectors/selectors.js

```diff
@@ -280,17 +283,36 @@ export const getMetaMaskAccounts = createSelector(
   getInternalAccounts,
   getMetaMaskAccountBalances,
   getMetaMaskCachedBalances,
-  (internalAccounts, balances, cachedBalances) =>
+  getMultichainBalances,
+  getMultichainNetwork,
+  (
+    internalAccounts,
+    balances,
+    cachedBalances,
+    multichainBalances,
+    multichainNetwork,
+  ) =>
     Object.values(internalAccounts).reduce((accounts, internalAccount) => {
       // TODO: mix in the identity state here as well, consolidating this
       // selector with `accountsWithSendEtherInfoSelector`
       let account = internalAccount;
 
-      if (balances[internalAccount.address]) {
+      if (
+        isEvmAccountType(internalAccount.type) ||
+        balances[internalAccount.address]
+      ) {
         account = {
           ...account,
           ...balances[internalAccount.address],
         };
+      } else {
+        account = {
+          ...account,
+          balance:
+            multichainBalances?.[internalAccount.id]?.[
+              MultichainNativeAssets[multichainNetwork.chainId]
+            ] ?? '0x0',
```

Comment:
I guess we miss a `.amount` here no? This should be:

```suggestion
            multichainBalances?.[internalAccount.id]?.[
              MultichainNativeAssets[multichainNetwork.chainId]
            ].amount ?? '0x0',
```